pkg/configurator/validating_webhook_test.go

```diff
@@ -258,9 +258,10 @@ func TestValidateFields(t *testing.T) {
 	assert.Nil(err)
 
 	testCases := []struct {
-		testName  string
-		configMap corev1.ConfigMap
-		expRes    *admissionv1.AdmissionResponse
```

Comment:
We should continue testing the admission response.

Reply:
Do you mean checking if the AdmissionResponse is created? I removed it because before I was using the AdmissionResponse to check the ReasonForDenial, but that check wasn't actually checking that.

Reply:
I am suggesting to keep the `&admissionv1.AdmissionResponse` object in the test case expectation and compare it with the result returned by `validateFields()`. That way, the entire response will be matched for equality which includes the reason as well.

Reply:
Ok, separated the test cases so that the `&admissionv1.AdmissionResponse` were comparable